src/main.rs

```diff
@@ -208,6 +210,25 @@ fn main() {
             import(&config, &branch, &message);
         }
 
+        "new" => {
+            if matches.free.len() == 2 {
+                let dest = matches.free[1].clone();
+
+                match create_new_project(&dest) {
+                    Ok(_) => info!("Created new project at {}", dest),
+                    Err(e) => {
+                        error!("{}", e);
+                        error!("Could not create a new cobalt project");
+                        std::process::exit(1);
+                    }
+                }
+            } else {
+                println!("cobalt new DIRECTORY");
```

Comment:
Or on second thought it's a good idea to print something here, but please make it an actual error message, I didn't really understand it at first. Something like "Please specify a project name".